aroma/utils.py

```diff
@@ -5,138 +5,67 @@
 
 import nibabel as nib
 import numpy as np
-from nilearn import image, masking
+from nilearn import masking
+from scipy import stats
+from tedana.decomposition import tedica, ma_pca
+from tedana.utils import get_spectrum
+from tedana.stats import computefeats2
 
+from .mixture import GGM
 
-def runICA(fsl_dir, in_file, out_dir, mel_dir_in, mask, dim, TR):
-    """Run MELODIC and merge the thresholded ICs into a single 4D nifti file.
+
+def runICA(in_file, mask, n_components, TR):
```

Comment:
@eurunuela @CesarCabelleroGaudes I would love to get your thoughts on this approach, even though it's not currently functional.